### ISSUE TITLE
[ic-top-navigation] - Landmark-unique accessibility violation

### ISSUE THREAD

**Issue by MI6-286** [mi6/ic-ui-kit#2690]
## Summary of the bug 
There is a landmark-unique accessibility violation when the top navigation menu is opened in mobile mode and the menu contains more than one navigation group.

## 🪜 How to reproduce 
1. Open a top navigation menu in mobile mode that contains more than one ic-navigation-group
2. Run accessibility fast pass
or
4. Run the a11y tests within that are currently commented out within IcNavigationMenu Cypress test file

## 📸 Screenshots or code 
Impacted area highlighted:
![image](https://github.com/user-attachments/assets/c8822cf3-7c4e-4d66-8e43-114df6f0fe70)

Details from storybook accessibility tab:
![image](https://github.com/user-attachments/assets/a0628f84-6bac-4de4-bd4c-e4d91db776d5)

## 🧐 Expected behaviour 
There should be no accessibility violations.

## Additional info 
Two a11y tests within Ic-Navigation-menu.cy.tsx should be uncommented when resolved.

**Comment by GCHQ-Developer-847** [comment]
Resolved as part of #3259 (PR #3378). The `<nav>`s have an aria-labelledby attribute to give them a unique name.
